src/features/emojiMod.ts

```diff
@@ -85,16 +98,17 @@ const reactionHandlers = {
       }
     }
   },
-  "👎": (bot, reaction, message, member) => {
-    if (cooldown.hasCooldown(member.id, "thumbsdown")) {
+  "👎": (reaction, message, member) => {
+    if (!message.guild || cooldown.hasCooldown(member.id, "thumbsdown")) {
       return;
     }
+
     cooldown.addCooldown(member.id, "thumbsdown", 60); // 1 minute
 
     const reactions = thumbsDownEmojis.reduce(
       (acc, emoji) => {
         if (message.reactions.cache.get(emoji)) {
-          acc.count += message.reactions.cache.get(emoji).count;
+          acc.count += message.reactions.cache.get(emoji)?.count || 0;
```

Comment:
Why is this `?.count`? Does TS not infer that it's not null here?

Reply:
Nope, I even tried moving the two cache calls to one variable but it still doesn't infer that it isn't null/undefined